src/utils/notifications-utils.js

```diff
@@ -0,0 +1,4 @@
+export function validateEmail(email) {
+  // eslint-disable-next-line no-useless-escape
+  return /^.+\@.+\..+$/.test(email)
```

Comment:
There's no real way to win with email validation... but we should at least be consistent :)

Let's use the same as in aragon/aragon: https://github.com/aragon/aragon/blob/master/src/utils.js#L148